scripts/sct_utils.py

```diff
@@ -12,16 +12,15 @@
 #
 # About the license: see the file LICENSE.TXT
 #########################################################################################
+import os
+import sys
```

Comment:
same here

Reply:
This one is a cleanup, `from sys import stdout` is not used anymore in the code.